spec/models/bookmark_search_form_spec.rb

```diff
@@ -91,6 +91,24 @@
     end
   end
 
+  describe "searching" do
+    let(:work1) { create(:posted_work, language_id: 1) }
+    let(:work2) { create(:posted_work, language_id: 2) }
+
+    let!(:bookmark1) { create(:bookmark, bookmarkable: work1) }
+    let!(:bookmark2) { create(:bookmark, bookmarkable: work2) }
+
+    before { run_all_indexing_jobs }
+
+    context "by work language" do
```

Comment:
Would it make sense to move this `context` block into the previous `describe "bookmarkable_search_results"` block, since it's also testing `bookmarkable_search_results`? 

The organization in this whole file is a little iffy, though, so it's not super important.